<<<<<<< HEAD
From 72f7d8abee34107b815e49540f2fad7f1ab0b2cc Mon Sep 17 00:00:00 2001
=======
From 8e6186be04e2819b6e3586e5d1aeb8a824e1979f Mon Sep 17 00:00:00 2001
>>>>>>> 903308adb4b (Improved error handling, differentiate nix/non-nix networks)
From: Simon Bruder <simon@sbruder.de>
Date: Thu, 25 Feb 2021 09:20:51 +0100
Subject: [PATCH] Skip testing restore with permission failure

The test fails in sandboxed builds.
---
<<<<<<< HEAD
 cmd/restic/cmd_restore_integration_test.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmd/restic/cmd_restore_integration_test.go b/cmd/restic/cmd_restore_integration_test.go
index 2c7cbe1fb..f25f13754 100644
--- a/cmd/restic/cmd_restore_integration_test.go
+++ b/cmd/restic/cmd_restore_integration_test.go
@@ -200,6 +200,7 @@ func TestRestoreLatest(t *testing.T) {
=======
 cmd/restic/integration_test.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmd/restic/integration_test.go b/cmd/restic/integration_test.go
index 7d198d33..1588ccb1 100644
--- a/cmd/restic/integration_test.go
+++ b/cmd/restic/integration_test.go
@@ -1170,6 +1170,7 @@ func TestRestoreLatest(t *testing.T) {
>>>>>>> 903308adb4b (Improved error handling, differentiate nix/non-nix networks)
 }
 
 func TestRestoreWithPermissionFailure(t *testing.T) {
+	t.Skip("Skipping testing restore with permission failure")
 	env, cleanup := withTestEnvironment(t)
 	defer cleanup()
 
-- 
<<<<<<< HEAD
2.41.0
=======
2.29.2
>>>>>>> 903308adb4b (Improved error handling, differentiate nix/non-nix networks)

